docs/site/Model.md

```diff
@@ -190,9 +188,197 @@ class Product extends Entity {
 }
 ```
 
-Additionally, the model decorator is able to build the properties object through
-the information passed in or inferred by the property decorators, so the
-properties key value pair can also be omitted.
+However, the model decorator in LoopBack 4 is not exactly the same as what it is
+in lb3. For example, in
```

Comment:
Nitpick: since you used `LoopBack 4` earlier in the sentence, maybe change `lb3` to `LoopBack 3` for consistency?